compiler/ballerina-lang/src/main/java/org/ballerinalang/compiler/CompilerOptionName.java

```diff
@@ -63,6 +63,7 @@ public enum CompilerOptionName {
     STICKY("sticky"),
 
     ENABLE_CACHE("enableCache"),
+    REMOTE_MANAGEMENT("remoteManagement"),
```

Comment:
Shall we change this to `enableRuntimeManagement` Also the CLI plugin

Reply:
Correct word is `enableRemoteManagement`

Reply:
So we need to set as `enableRemoteManagement = true` under build-options of the `Ballerina.toml` file. For the observability, we used `observabilityIncluded`. In order to comply with that pattern, then wouldn't it be something like `remoteManagementEnabled`?

Reply:
After offline discussion, we decided to go ahead with `remoteManagement`.